exercises/sum.cpp

```diff
@@ -6,3 +6,16 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+using namespace std;
+
+int sum(int a, int b) {
+  return a + b;
+}
+
+int main(int argc, char const *argv[]) {
+  cout << sum(1, 2) << endl;
+  return 0;
```

Comment:
Potresti rendere il codice più leggibile e al posto di `return 0` potresti fare un `#DEFINE EXIT_SUCCESS 0` in alto e nel return scrivere `return EXIT_SUCCESS;`

Reply:
Fatto